### PR TITLE
Update pstn-usage-report.md

### PR DESCRIPTION
**Pull request by SanderSade**
Note that a bigger update of documentation is coming, this change only has some minor changes related to bugs or incidents.

* Direct routing export:

- clarify that CorrelationId is not unique
- add info about AAD Object field
- SIP Address was renamed to UPN to correctly reflect the content of the field

* Added Microsoft Graph PSTN and DR report links.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-analytics-and-reports/pstn-usage-report.md [hunk 1]
```diff
@@ -159,3 +160,5 @@ You can export data up to five months (150 days) from the current date unless co
 ## Related topics
 
 - [Teams analytics and reporting](teams-reporting-reference.md)
+- [PSTN call report in Microsoft Graph](https://docs.microsoft.com/en-us/graph/api/callrecords-callrecord-getpstncalls?view=graph-rest-1.0&tabs=http)
+- [Direct routing report in Microsoft Graph](https://docs.microsoft.com/en-us/graph/api/callrecords-callrecord-getdirectroutingcalls?view=graph-rest-1.0&tabs=http)
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/pstn-usage-report.md:164]:
```suggestion
- [PSTN call report in Microsoft Graph](/graph/api/callrecords-callrecord-getpstncalls?view=graph-rest-1.0&tabs=http)
- [Direct routing report in Microsoft Graph](/graph/api/callrecords-callrecord-getdirectroutingcalls?view=graph-rest-1.0&tabs=http)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SanderSade, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by SanderSade** [comment]
@scanum , @SerdarSoysal gentle ping, it has been over two weeks since approval.

**Pull request merged**